packages/fakta-omsorg/src/components/OmsorgInfoPanel.tsx

```diff
@@ -63,33 +61,34 @@ export const OmsorgInfoPanel = ({
         </AksjonspunktHelpTextHTML>
       )}
       <PersonopplysningerForFamilie alleKodeverk={alleKodeverk} personoversikt={personoversikt} />
-      <Form
-        formMethods={formMethods}
-        onSubmit={(values: FormValues) => submitCallback(transformValues(values))}
-        setDataOnUnmount={setFormData}
-      >
-        <VStack gap="6">
-          <OmsorgFaktaFields
-            readOnly={readOnly}
-            aksjonspunkter={aksjonspunkter}
-            alleMerknaderFraBeslutter={alleMerknaderFraBeslutter}
-          />
-          <FaktaBegrunnelseTextFieldNew
-            isSubmittable={submittable}
-            isReadOnly={readOnly}
-            hasBegrunnelse={true}
-            hasVurderingText
-          />
-          <div>
-            <FaktaSubmitButtonNew
+      {harAksjonspunkt && (
+        <Form
+          formMethods={formMethods}
+          onSubmit={(values: FormValues) => submitCallback(transformValues(values))}
+          setDataOnUnmount={setFormData}
+        >
+          <VStack gap="6">
+            {harAksjonspunkt && (
+              <OmsorgFaktaFields readOnly={readOnly} alleMerknaderFraBeslutter={alleMerknaderFraBeslutter} />
+            )}
+
+            <FaktaBegrunnelseTextField
               isSubmittable={submittable}
               isReadOnly={readOnly}
-              isSubmitting={formMethods.formState.isSubmitting}
```

Comment:
Sikkert ikkje feil å fjerna desse altso, men blir det korrekt oppførsel? Må ein ikkje ha isSubmitting for å hindra at knappen er trykkbar etter at ein har trykka? Og for å visa spinning-ikon?

Fungerar ikkje dei fleste panela sånn at bekreft-knapp er trykkbar først etter at form er dirty? Trur litt av greia er å hindra at ein går tilbake til panelet og bekreftar på nytt utan å endra data.

Reply:
Diffen her viser endringene litt rart, ser ut som det er knappen som er endret, men akkurat den linjen er tekstfeltet. Knappen har fortsatt isSubmitting 😄